manual/src/cmds/debugger.etex

```diff
@@ -79,6 +79,41 @@ file before giving control to the user. The default file is
 ".ocamldebug" in the current directory if it exists, otherwise
 ".ocamldebug" in the user's home directory.
 
+\subsection{ss:debugger-environ}{Environment Variables}
+
+The debugger supports the "BUILD_PATH_PREFIX_MAP" environment
+variable, as defined by the ``BUILD_PATH_PREFIX_MAP specification''
+(see
+\url{https://reproducible-builds.org/specs/build-path-prefix-map}).
+
+When the product being debugged was built, ``BUILD_PATH_PREFIX_MAP''
+was set by the build tools so that build-time absolute paths
+where mapped to abstract reproducible paths.
+
+Before ocamldebug is invoked, ``BUILD_PATH_PREFIX_MAP`` must again
+be set, but this time to cause the logical inverse of the
```

Comment:
The phrasing of this paragraph suggests that setting BUILD_PATH_PREFIX_MAP is mandatory. The documentation should be clear that this is an optional feature.

Reply:
Done.